<?xml version="1.0"?>
<launch>

  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />

  <node pkg="tf" type="static_transform_publisher" name="table_static_tf" args="1 -0.5 -0.2 0 0 0.7071 0.7071  camera_link table 100" />
  <!-- Launch pick and place -->
  <node name="kinect_pointing" launch-prefix="$(arg launch_prefix)" pkg="art_table_pointing" 
	type="art_table_pointing_kinect" output="screen" respawn="false">
    <param name="x_offset" value="0"/>
    <param name="y_offset" value="0"/>
    <param name="table_width" value="1"/>
    <param name="table_height" value="0.8"/>
    <param name="table_frame" value="/table"/>
    <param name="show_arrows" value="true"/>

  </node>
  
  

</launch>
